util/markov.py

```diff
@@ -0,0 +1,69 @@
+from markovchain import JsonStorage
+from markovchain.text import MarkovText, ReplyMode
+import json
+
+import markovify
+
+
+def markovchain_example():
```

Comment:
Just keeping the examples for now in case we run into problems and need to try the other library